docs/src/pages/system/basics/basics.md

```diff
@@ -33,318 +23,339 @@ npm install @material-ui/system
+  If you are already using the core components, it comes with no extra overhead.
 
-{{"demo": "pages/system/basics/StyledComponents.js", "defaultCodeOpen": true}}
+Cons:
 
-### Emotion
+- The runtime performance take a hit.
 
-{{"demo": "pages/system/basics/Emotion.js", "defaultCodeOpen": true}}
+  | Benchmark case                    | Code snippet         | Time normalized |
+  | :-------------------------------- | :------------------- | --------------- |
+  | a. Render 1,000 primitives        | `<div className="">` | 100ms           |
+  | b. Render 1,000 components        | `<Div>`              | 110ms           |
+  | c. Render 1,000 styled components | `<StyledDiv>`        | 160ms           |
+  | d. Render 1,000 Box               | `<Box sx={}>`        | 270ms           |
 
-## Responsive
+  _Head to the [benchmark folder](https://github.com/mui-org/material-ui/tree/next/benchmark/browser) for a reproduction of these metrics._
 
-**All** the props are responsive. There are three different APIs – Array, Object, and Collocation – that each use this default (but customizable) breakpoints theme structure:
+  We believe that for most applications, it's **fast enough**. There are simple workarounds when performance becomes critical. For instance, when rendering a list with many items, you can use a CSS child selector to have a single "style injection" point (using d. for the wrapper and a. for each item).
 
-```js
-const values = {
-  xs: 0,
-  sm: 600,
-  md: 960,
-  lg: 1280,
-  xl: 1920,
-};
+## Usage
 
-const theme = {
-  breakpoints: {
-    keys: ['xs', 'sm', 'md', 'lg', 'xl'],
-    up: (key) => `@media (min-width:${values[key]}px)`,
-  },
-};
-```
+### Design tokens in the theme
 
-### Array
+You can explore the [System properties](/system/properties/) page to discover how the different CSS (and custom) properties are mapped to the theme keys.
 
-```jsx
-<Box p={[2, 3, 4]} />
-
-/**
- * Outputs:
- *
- * padding: 16px;
- * @media (min-width: 600px) {
- *   padding: 24px;
- * }
- * @media (min-width: 960px) {
- *   padding: 32px;
- * }
- */
-```
+### Shorthands
 
-### Object
+There are lots of shorthands available for the CSS properties.
+These are documented in the next pages, for instance, [the spacing](/system/spacing/).
+Here are an example leveraging them:
 
 ```jsx
-<Box p={{ xs: 2, sm: 3, md: 4 }} />
-
-/**
- * Outputs:
- *
- * padding: 16px;
- * @media (min-width: 600px) {
- *   padding: 24px;
- * }
- * @media (min-width: 960px) {
- *   padding: 32px;
- * }
- */
+<Box
+  sx={{
+    boxShadow: 1, // theme.shadows[1]
+    color: 'primary.main', // theme.palette.primary.main
+    m: 1, // margin: theme.spacing(1)
+    p: {
+      sx: 1, // [theme.breakpoints.up('sx')]: : { padding: theme.spacing(1) }
+    },
+    zIndex: 'tooltip', // theme.zIndex.tooltip
+  }}
+>
```

Comment:
It might be simpler just to list some examples of CSS property shorthands. This is example is showing both CSS property and theme value accessors, which makes the sentence that follows confusing.

Reply:
I think that both `mt` instead `marginTop` and `marginTop: 1` instead of `marginTop: (theme) => theme.spacing(1)` can be considered shorthands.

Reply:
Yes, both properties, _and_ values. If you're going to skip this part, and use CSS, you might as well just skip the whole Style System section, and use inline styles.

Reply:
Sorry I don’t understand the point, should we list only the properties shorthands?

Reply:
> Yes, both properties, and values. If you're going to skip this part, and use CSS, you might as well just skip the whole Style System section, and use inline styles.

I don't understand the logic in this conclusion.